### PR TITLE
Gip tenant setup

### PR DESCRIPTION
**Pull request by jwade1327**

### CODE REVIEW

File: tenants/gip/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ac-business-media/gip",
+  "version": "0.0.0",
+  "author": "Jessica Wade <jessica.wade@parameter1.com>",
+  "repository": "https://github.com/parameter1/newsletter-ac-business-media/tree/master/tenants/fcp",
```
Comment (brandonbk) [inline tenants/gip/package.json:5]: Change fcp to gip here

File: tenants/gip/config/getFooterData.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable consistent-return */
```
Comment (brandonbk) [inline tenants/gip/config/getFooterData.js:1]: I don't believe this file is needed anymore; could you confirm either way?

File: tenants/gip/components/ad-unit.marko [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import emailX from "../config/email-x";
```
Comment (brandonbk) [inline tenants/gip/components/ad-unit.marko:1]: This component, if used, should be moved out of fcp and gip and into the shared package

File: tenants/gip/components/banner-element.marko [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { getAsObject } from "@parameter1/base-cms-object-path";
```
Comment (brandonbk) [inline tenants/gip/components/banner-element.marko:1]: This component, if used, should be moved into the shared package.

**Review by brandonbk — CHANGES_REQUESTED** [review]

File: tenants/gip/package.json (hunk 1 above)
Comment (jwade1327) [reply tenants/gip/package.json]: Updated this change

File: tenants/gip/config/getFooterData.js (hunk 2 above)
Comment (jwade1327) [reply tenants/gip/config/getFooterData.js]: This file as been removed as well as config/footers

File: tenants/gip/components/banner-element.marko (hunk 4 above)
Comment (jwade1327) [reply tenants/gip/components/banner-element.marko]: @brandonbk all of these changes have been made

**Pull request closed without merging**